app/controllers/api/verify/document_capture_controller.rb

```diff
@@ -1,88 +0,0 @@
-module Api
-  module Verify
-    class DocumentCaptureController < BaseController
-      include ApplicationHelper
-      include EffectiveUser
-
-      def create
-        result = Idv::ApiDocumentVerificationForm.new(
-          verify_params,
-          analytics: analytics,
-          irs_attempts_api_tracker: irs_attempts_api_tracker,
-          flow_path: params[:flow_path],
-        ).submit
-
-        if result.success?
-          enqueue_job
-
-          render json: { success: true, status: 'in_progress' }, status: :accepted
-        else
-          render_errors(result.errors)
-        end
-      end
-
-      private
-
-      def enqueue_job
-        verify_document_capture_session = DocumentCaptureSession.
-          find_by(uuid: params[:document_capture_session_uuid])
-        verify_document_capture_session.requested_at = Time.zone.now
-        verify_document_capture_session.create_doc_auth_session
-
-        applicant = {
-          user_uuid: effective_user.uuid,
-          uuid_prefix: current_sp&.app_id,
-          document_arguments: document_attributes,
-        }
-        Idv::Agent.new(applicant).proof_document(
```

Comment:
Should we get rid of `Agent#proof_document`, as well as the `DocumentProofingJob` class?